lib/xtool/xtcore.cpp

```diff
@@ -156,7 +156,7 @@ int main(int argc, char *argv[])
                 } else if (value == "any") {
                     autoconnectGameID = -1;
                 } else {
-                    autoconnectGameID = strtol(argv[i], &argv[i], 0);
+                    autoconnectGameID = strtol(argv[i], NULL, 0);

```

Comment:
Perhaps missed cast to `unsigned short` (i don't know, the code above includes that).

Reply:
well I see it is a signed int here :)

Reply:
Ah, my fail! Ok!